testsuite/integration-arquillian/servers/auth-server/quarkus/pom.xml

```diff
@@ -188,6 +188,31 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>includeProprietaryDependencies</id>
```

Comment:
I might be wrong but I believe we need to use a unique ID for each execution, otherwise it will overwrite the config of the previously defined executions with the same ID. And this ID is already used by the `product` profile.

Reply:
Adressed.